examples/test/local/test_bert_cuda_gpu.py

```diff
@@ -15,6 +15,7 @@ def setup():
     os.chdir(get_example_dir("bert"))
 
 
+@pytest.mark.skip(reason="Disable failing tests")
```

Comment:
these tests don't fail in the main ci pipeline so I don't think we should skip them. For the nightly pipelines, once we move to docker runs, we should be able to fix them too.

Reply:
As per my discussion with @devang-ml, the decision is to disable any and all failing tests and get green build. I will send out a list of all disabled tests which will be assigned to developers to fix and enable while keeping all lights green.

The changes in this PR makes all 4 Olive pipelines green.

Reply:
I see. thanks for the info.